pkg/network/protocols/http/monitor.go

```diff
@@ -185,6 +186,7 @@ func (m *Monitor) Stop() {
 
 func (m *Monitor) process(data []byte) {
 	tx := (*ebpfHttpTx)(unsafe.Pointer(&data[0]))
+	log.Infof("HTTP_TX: %+v", tx)
```

Comment:
do we want it to be printed? if so, I think `trace` will be suitable here

Reply:
argh, good catch. I'm going to remove this altogether :sweat_smile: